datafusion/common/src/error.rs

```diff
@@ -125,7 +125,7 @@ pub enum SchemaError {
     /// No field with this name
     FieldNotFound {
         field: Column,
-        valid_fields: Option<Vec<Column>>,
+        valid_fields: Vec<Column>,
```

Comment:
I've not found any reason to have `Option` here, `Vec` could be empty and it would be zero-size. Moreover, there is no usages of `None` at all.